src/overlays/actors/ovl_Bg_Bombwall/z_bg_bombwall.c

```diff
@@ -15,16 +15,41 @@ void BgBombwall_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgBombwall_Update(Actor* thisx, GlobalContext* globalCtx);
 void BgBombwall_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-void func_8086E7D0(BgBombwall* this, GlobalContext* globalCtx);
-void func_8086EAC0(BgBombwall* this, GlobalContext* globalCtx);
+// void BgBombwall_InitDynapoly(BgBombwall* this, GlobalContext* globalCtx);
+void BgBombwall_DestroyInternal(BgBombwall* this, GlobalContext* globalCtx);
```

Comment:
you can remove this one thats commented out. and if anymore of these are not required then you can remove them as well. we only forward declare when necessary